### ISSUE TITLE
kOS Doesn't re-enable commands when a previous lost RT connection gets re-established.

### ISSUE THREAD

**Issue by Dunbaratu** [KSP-KOS/KOS#906]
Once you lose your RT connection and the Terminal issues you this message:
```
------------------------------------------
Signal lost.  Waiting to re-acquire signal.
```
Then you're locked out forever.  It won't bring the status back when you have connection again.

To recreate:

From a new sandbox game without any RT network established yet, put a probe vessel in low Kerbin orbit, with an omni antenna to talk to KSC, so you can only get KSC connection when above KSC.  Time warp to just BEFORE the point where you'd lose signal, and then drop from time warp back to 1x speed so the CPU is active and running when the signal loss happens.

You get the "signal lost" message, and nothing you type will execute, as expected.  This is normal so far.

But then time warp around one orbit, coming back over the KSC again so the map view shows the green line connecting you to the KSC again.  The terminal is still showing the "signal lost" message.  You are still being locked out even though signal is back.

Note, as an ugly workaround, if you force the game to unload and then reload the vessel by switching to another vessel and coming back, or by switching to KSC and then coming back, then this forced reloading of the vessel will get it un-stuck from this mode.

But that's the only thing that unsticks it.  Even if you toggle power on the kOS part, that doesn't reset the 'lost signal' status.

Note, RT is allowing manual flight control at this point, so signal is definitely working again.

(and yes, I have CONFIG:RT set to true).

**Comment by ianjirka** [comment]
I can confirm the same behavior even if the vessel also is crewed.  RT reports 'local control' but the kOS console is dead because of lost signal.

**Comment by Dunbaratu** [comment]
As an entirely seperate issue, and perhaps it's an RT issue not kOS, I really wish a crewed vessel had access to its kOS terminal consoles even when out of RT signal range - after all, there's a crew member in there to type commands out.

**Comment by tomekpiotrowski** [comment]
I was not able to reproduce this with the current version of kOS. I've tested it in few scenarios including exactly the one described above. 'Signal lost' status disappears as it should and any pending commands get executed properly. It must have been fixed by one of the previous RT PRs.

**Comment by hvacengi** [comment]
I have tested this issue as well and can confirm that this bug appears to have been handled.  I tested using the procedure prescribed above.  I also made sure that everything functions properly if you are in timewarp at either transition.  The tests were performed using the soon to be released version of kOS and Remote Tech version 1.6.8.

I'm going to close the issue, if some one sees this issue again in the future we can reopen this, or create a new issue.
